src/Core/Prefab/Globe/Atmosphere.js

```diff
@@ -65,6 +65,7 @@ function Atmosphere() {
         side: THREE.FrontSide,
         blending: THREE.AdditiveBlending,
         transparent: true,
+        depthWrite: false,
```

Comment:
The atmosphere can hide transparent object if its frontier is between the camera and the object.